datafusion/core/src/datasource/file_format/parquet.rs

```diff
@@ -227,6 +230,17 @@ impl ParquetFormat {
     pub fn options(&self) -> &TableParquetOptions {
         &self.options
     }
+
+    /// Return `true` if should use view types.
```

Comment:
Also, what do you think about calling this `force_view_types` rather than `should_use_view_types`?

Likewise then the setter would be named `with_force_view_types`

Reply:
renamed and docs added.